src/sbm.jl

```diff
@@ -1049,3 +1052,35 @@ function update_after_subsurfaceflow(sbm::SBM, zi, exfiltsatwater)
         sbm.zi[i] = zi[i]
     end
 end
+
+function update_total_water_storage(sbm::SBM, rnet, rrouting, lrouting)
```

Comment:
Perhaps good to add some docstring to this function to explain what it does, and what input it gets. I would also opt for using more detailed variables names instead of abbreviations, to make the code easier to understand. For example, `rnet` is not very clear to me, unless I look at what variable you pass when you call the function.

Reply:
I'll change it.